SpatialGDK/Source/SpatialGDKEditor/Private/WorkerTypeCustomization.cpp

```diff
@@ -18,16 +18,12 @@ void FWorkerTypeCustomization::CustomizeHeader(TSharedRef<class IPropertyHandle>
 	if (WorkerTypeNameProperty->IsValidHandle())
 	{
 		HeaderRow.NameContent()
-			[
-				StructPropertyHandle->CreatePropertyNameWidget()
-			]
-		.ValueContent()
-			[
-				PropertyCustomizationHelpers::MakePropertyComboBox(WorkerTypeNameProperty,
-				FOnGetPropertyComboBoxStrings::CreateStatic(&FWorkerTypeCustomization::OnGetStrings),
-				FOnGetPropertyComboBoxValue::CreateStatic(&FWorkerTypeCustomization::OnGetValue, WorkerTypeNameProperty),
-				FOnPropertyComboBoxValueSelected::CreateStatic(&FWorkerTypeCustomization::OnValueSelected, WorkerTypeNameProperty))
-			];
+			[StructPropertyHandle->CreatePropertyNameWidget()]
```

Comment:
Was there a comment to disable formatting for a specific block? This looks like one of the places we might want that.